force-app/test/ACCT_AccountMerge_TEST.cls

```diff
@@ -197,7 +197,7 @@ private with sharing class ACCT_AccountMerge_TEST {
         System.assertEquals(cHH * cCon, listCon.size());
                     
         // verify Addresses are merged
-        System.assertEquals(cHH + 1, [SELECT count() FROM Address__c]);
+        System.assertEquals(cHH * 2, [SELECT count() FROM Address__c]);
```

Comment:
Why check for double (2x) the number of records here; which I think would be 6 address records? The previous assert was checking for only 4 records. Does the new override cause more address records to be created?

Reply:
@force2b thank you for your feedback. The assert now checks for double the number of address records because the new Address Override functionality creates a separate address for each contact with the override enabled. Previously, the contacts shared the household address, resulting in fewer records. With the override, each contact now gets its own distinct address, which correctly leads to a higher count. This behavior aligns with the goal of maintaining independent addresses for contacts with Address Override enabled.